# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2015-07-22 16:11+0800\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: account/forms.py:15
msgid "email address format error"
msgstr "电子邮件地址格式不正确"

#: account/models.py:16
msgid "Mobile"
msgstr "手机号"

#: account/models.py:17
msgid "User Type"
msgstr "用户类型"

#: account/models.py:26 account/models.py:27
msgid "UserProfile"
msgstr "用户设置"

#: account/models.py:69
msgid "Contract name"
msgstr "合同名称"

#: account/models.py:70
msgid "Customer name"
msgstr "客户名称"

#: account/models.py:71
msgid "Start Date"
msgstr "开始日期"

#: account/models.py:72
msgid "End Date"
msgstr "结束日期"

#: account/models.py:73 account/models.py:96 backup/models.py:30
#: backup/models.py:139 firewall/models.py:15 firewall/models.py:36
#: floating/models.py:29 forum/models.py:11 forum/models.py:28
#: instance/models.py:44 lbaas/models.py:23 lbaas/models.py:46
#: lbaas/models.py:76 lbaas/models.py:104 network/models.py:15
#: network/models.py:56 network/models.py:74 network/models.py:90
#: volume/models.py:17
msgid "Deleted"
msgstr "已删除"

#: account/models.py:74 account/models.py:97 account/models.py:115
#: backup/models.py:28 backup/models.py:137 firewall/models.py:14
#: firewall/models.py:37 floating/models.py:27 forum/models.py:12
#: forum/models.py:29 image/models.py:25 instance/models.py:26
#: instance/models.py:82 lbaas/models.py:22 lbaas/models.py:45
#: lbaas/models.py:75 lbaas/models.py:103 network/models.py:16
#: network/models.py:57 network/models.py:75 volume/models.py:16
msgid "Create Date"
msgstr "创建日期"

#: account/models.py:75 account/models.py:98
msgid "Update Date"
msgstr "修改日期"

#: account/models.py:88 account/models.py:89 account/settings.py:30
msgid "Contract"
msgstr "合同"

#: account/models.py:94
msgid "Resouce"
msgstr "资源"

#: account/models.py:95
msgid "Limit"
msgstr "配额"

#: account/models.py:102 account/models.py:103
msgid "Quota"
msgstr "配额"

#: account/models.py:110 backup/models.py:125 floating/models.py:22
msgid "Resource"
msgstr "资源"

#: account/models.py:111 backup/models.py:121
msgid "Resource ID"
msgstr "资源 ID"

#: account/models.py:112 backup/models.py:127
msgid "Resource Name"
msgstr "资源名称"

#: account/models.py:113
msgid "Action"
msgstr "操作"

#: account/models.py:114
msgid "Result"
msgstr "结果"

#: account/models.py:137
#, python-format
msgid "%(resource)s:%(resource_name)s execute %(action)s operation"
msgstr "%(resource)s:%(resource_name)s 执行了 %(action)s 操作"

#: account/models.py:155 account/models.py:156
msgid "Operation"
msgstr "操作日志"

#: account/models.py:175
msgid "just now"
msgstr "刚刚"

#: account/models.py:178
#, python-format
msgid "%(minutes)d minutes ago"
msgstr "%(minutes)d分钟前"

#: account/models.py:181
#, python-format
msgid "%(hours)d hours ago"
msgstr "%(hours)d小时前"

#: account/models.py:184
#, python-format
msgid "%(days)d days ago"
msgstr "%(days)d天前"

#: account/models.py:187
#, python-format
msgid "%(years)d years ago"
msgstr "%(years)d年前"

#: account/models.py:204
msgid "Notification"
msgstr "通知"

#: account/models.py:205
msgid "Notifications"
msgstr "通知"

#: account/settings.py:7
msgid "Personal"
msgstr "个人"

#: account/settings.py:8
msgid "Company"
msgstr "企业"

#: account/settings.py:12 account/settings.py:22 backup/models.py:22
#: instance/models.py:52 instance/models.py:53
msgid "Instance"
msgstr "云主机"

#: account/settings.py:13
msgid "CPU"
msgstr "CPU"

#: account/settings.py:14
msgid "Memory(MB)"
msgstr "内存(MB)"

#: account/settings.py:15
msgid "Volume(GB)"
msgstr "磁盘(GB)"

#: account/settings.py:16
msgid "Volume Count"
msgstr "磁盘数量"

#: account/settings.py:17
msgid "Floating IP"
msgstr "公网IP"

#: account/settings.py:23 account/settings.py:27 volume/models.py:29
#: volume/models.py:30
msgid "Volume"
msgstr "磁盘"

#: account/settings.py:24 instance/models.py:35 network/models.py:36
#: network/models.py:37 network/models.py:89
msgid "Network"
msgstr "私有网络"

#: account/settings.py:25 network/models.py:61 network/models.py:62
msgid "Subnet"
msgstr "子网"

#: account/settings.py:26 network/models.py:80 network/models.py:81
msgid "Router"
msgstr "路由器"

#: account/settings.py:28
msgid "Firewall"
msgstr "防火墙"

#: account/settings.py:29
msgid "FirewallRules"
msgstr "防火墙规则"

#: account/settings.py:35
msgid "reboot"
msgstr "重启"

#: account/settings.py:36
msgid "power_on"
msgstr "开机"

#: account/settings.py:37
msgid "power_off"
msgstr "关机"

#: account/settings.py:38
msgid "vnc_console"
msgstr "VNC登录"

#: account/settings.py:39
msgid "bind_floating"
msgstr "绑定公网IP"

#: account/settings.py:40
msgid "unbind_floating"
msgstr "解绑公网IP"

#: account/settings.py:41
msgid "change_firewall"
msgstr "更新防火墙"

#: account/settings.py:42
msgid "attach_volume"
msgstr "挂载磁盘"

#: account/settings.py:43
msgid "detach_volume"
msgstr "卸载磁盘"

#: account/settings.py:44
msgid "terminate"
msgstr "删除"

#: account/settings.py:45
msgid "launch"
msgstr "创建"

#: account/settings.py:46
msgid "create"
msgstr "创建"

#: account/settings.py:47
msgid "update"
msgstr "更新"

#: account/settings.py:48
msgid "delete"
msgstr "删除"

#: account/settings.py:49
msgid "attach router"
msgstr "连接路由器"

#: account/settings.py:50
msgid "detach router"
msgstr "离开路由器"

#: account/settings.py:63
#, fuzzy
#| msgid "Operation"
msgid "Information"
msgstr "操作日志"

#: account/settings.py:64
msgid "Success"
msgstr ""

#: account/settings.py:65
msgid "Error"
msgstr ""

#: account/settings.py:66
msgid "Warning"
msgstr ""

#: account/settings.py:67
msgid "Danger"
msgstr ""

#: account/views.py:168
msgid "Contract is created successfully!"
msgstr "合同创建成功！"

#: account/views.py:172
msgid "Contract data is not valid!"
msgstr "合同数据不对！"

#: account/views.py:177
msgid "Failed to create contract for unknown reason."
msgstr "未知错误导致合同创建失败."

#: account/views.py:195
msgid "Contract is updated successfully!"
msgstr "合同更新成功！"

#: account/views.py:200
msgid "Failed to update contract for unknown reason."
msgstr "未知错误导致合同更新失败."

#: account/views.py:217
msgid "Contracts have been deleted!"
msgstr "合同删除成功！"

#: account/views.py:221
msgid "Failed to delete contracts for unknown reason."
msgstr "未知错误导致合同删除失败．"

#: account/views.py:250
msgid "User has been deactivated!"
msgstr "用户已禁用，将不能登录"

#: account/views.py:261
msgid "User has been activated!"
msgstr "用户已激活，可以正常使用"

#: account/views.py:272
msgid "The new password doesn't match confirm password!"
msgstr "两次密码输入密码不匹配"

#: account/views.py:277
msgid "The user is not founded!"
msgstr "没有该用户！"

#: account/views.py:280
msgid "The original password is not correct!"
msgstr "原始密码不对！"

#: account/views.py:285
msgid "Password has been changed!"
msgstr "密码修改成功"

#: account/views.py:335
msgid "Quotas have been saved successfully!"
msgstr "配额已保存成功！"

#: account/views.py:339
msgid "Failed to save quotas for unknown reason."
msgstr "未知错误导致配额保存失败．"

#: account/views.py:357
msgid "Quota have been saved successfully!"
msgstr "配额保存成功！"

#: account/views.py:362
msgid "Failed to save quota for unknown reason."
msgstr "未知错误导致配额保存失败．"

#: account/views.py:371
msgid "Quota have been deleted successfully!"
msgstr "配额删除成功"

#: account/views.py:375
msgid "Failed to create quota for unknown reason."
msgstr "配额创建失败"

#: account/views.py:402 account/views.py:415
msgid "Notification is sent successfully!"
msgstr "消息发布成功！"

#: backup/models.py:16 idc/models.py:14 instance/models.py:80
msgid "Name"
msgstr "名称"

#: backup/models.py:17
msgid "Back Type"
msgstr "备份类型"

#: backup/models.py:20
msgid "Volumes"
msgstr "磁盘"

#: backup/models.py:29 backup/models.py:138 floating/models.py:28
msgid "Delete Date"
msgstr "删除日期"

#: backup/models.py:31 backup/models.py:140 floating/models.py:17
#: instance/models.py:23 lbaas/models.py:21 lbaas/models.py:44
#: lbaas/models.py:73 network/models.py:11 network/models.py:53
#: network/models.py:71 volume/models.py:15
msgid "Status"
msgstr "状态"

#: backup/models.py:112 backup/models.py:113
msgid "Backup"
msgstr "备份"

#: backup/models.py:123
msgid "Backend UUID"
msgstr ""

#: backup/models.py:129
msgid "Size(GB)"
msgstr ""

#: backup/models.py:132
msgid "RBD Image"
msgstr "镜像 "

#: backup/models.py:145 backup/models.py:146
msgid "Backup Item"
msgstr "备份项"

#: backup/settings.py:9
msgid "BACKUP_TYPE_FULL"
msgstr "全量备份"

#: backup/settings.py:10
msgid "BACKUP_TYPE_INCREMENT"
msgstr "增量备份"

#: backup/settings.py:27 backup/settings.py:40
msgid "Backup Waiting"
msgstr "等待备份"

#: backup/settings.py:28 backup/settings.py:41
msgid "Backup Available"
msgstr "可用"

#: backup/settings.py:29 backup/settings.py:42
msgid "Backup Error"
msgstr "错误"

#: backup/settings.py:30 backup/settings.py:43
msgid "Backup Deleted"
msgstr "已删除"

#: backup/settings.py:31 backup/settings.py:44
msgid "Backup Backuping"
msgstr "备份中"

#: backup/settings.py:32 backup/settings.py:45
msgid "Backup Restoring"
msgstr "恢复中"

#: backup/settings.py:33 backup/settings.py:46
msgid "Backup Pending Delete"
msgstr "待删除"

#: backup/settings.py:34 backup/settings.py:47
msgid "Backup Pending Restore"
msgstr "待恢复"

#: backup/settings.py:35 backup/settings.py:48
msgid "Backup Deleting"
msgstr "删除中"

#: firewall/models.py:8
msgid "Firewall Name"
msgstr "防火墙名称"

#: firewall/models.py:9
msgid "OS Firewall UUID"
msgstr "Firewall UUID"

#: firewall/models.py:10
msgid "Firewall desc"
msgstr "防火墙描述"

#: firewall/models.py:11 firewall/models.py:33 network/models.py:12
#: network/models.py:76
msgid "Default"
msgstr "默认"

#: firewall/models.py:24
msgid "OS Firewall Rules UUID"
msgstr ""

#: firewall/models.py:25
msgid "Direction"
msgstr "流向"

#: firewall/models.py:26
msgid "Ether type"
msgstr ""

#: firewall/models.py:27
msgid "Port range min"
msgstr ""

#: firewall/models.py:28
msgid "Port range max"
msgstr ""

#: firewall/models.py:29 lbaas/models.py:66
msgid "Protocol"
msgstr ""

#: firewall/models.py:30
msgid "remote group id UUID"
msgstr ""

#: firewall/models.py:31
msgid "remote ip prefix"
msgstr ""

#: firewall/settings.py:9
msgid "Ingress"
msgstr "下行"

#: firewall/settings.py:10
msgid "Egress"
msgstr "上行"

#: firewall/settings.py:17
msgid "IPv4"
msgstr "IPv4"

#: firewall/settings.py:18
msgid "IPv6"
msgstr "IPv6"

#: firewall/views.py:35
msgid "Creating firewall"
msgstr "创建防火墙"

#: firewall/views.py:39
msgid "Firewall create error"
msgstr "创建"

#: firewall/views.py:41 network/views.py:52 volume/views.py:63
msgid "Data valid error"
msgstr "数据格式错误"

#: firewall/views.py:48
msgid "Not selected Firewall"
msgstr "选择防火墙"

#: firewall/views.py:51
msgid "Firewall not exists"
msgstr "防火墙不存在"

#: firewall/views.py:53
msgid "Firewall default can not delete"
msgstr "默认防火墙允许删除"

#: firewall/views.py:55
msgid "Firewall used can not delete"
msgstr "被使用的防火墙不允许删除"

#: firewall/views.py:62
msgid "Firewall delete error"
msgstr "防火墙删除失败"

#: firewall/views.py:69
msgid "Firewall delete success"
msgstr "防火墙删除成功"

#: firewall/views.py:85
msgid "no selected security group"
msgstr "选择安全组"

#: firewall/views.py:95
msgid "Create firewall rule error"
msgstr "创建规则失败"

#: firewall/views.py:96
msgid "Create firewall rule success"
msgstr "创建规则成功"

#: firewall/views.py:97
msgid "Valid firewall rule nopass"
msgstr "规则格式错误"

#: firewall/views.py:103
msgid "Not selected Firewall rule"
msgstr "选择规则"

#: firewall/views.py:106
msgid "Firewall rule not exists"
msgstr "规则不存在"

#: firewall/views.py:112
msgid "Firewall rule delete success"
msgstr "规则删除成功"

#: firewall/views.py:115
msgid "Firewall rule delete error"
msgstr "规则删除失败"

#: floating/models.py:13 instance/models.py:43
msgid "Public IP"
msgstr "公网IP"

#: floating/models.py:19
msgid "Bandwidth MB"
msgstr "带宽 MB"

#: floating/models.py:23
msgid "Resource type"
msgstr "资源类型"

#: floating/models.py:51 floating/models.py:52
msgid "Floating"
msgstr "公网IP"

#: floating/settings.py:15 floating/settings.py:27
msgid "Floating ERROR"
msgstr "错误"

#: floating/settings.py:16 floating/settings.py:28
msgid "Floating Allocate"
msgstr "分配中"

#: floating/settings.py:17 floating/settings.py:29
msgid "Floating Avaiable"
msgstr "可用"

#: floating/settings.py:18 floating/settings.py:30
msgid "Floating Binded"
msgstr "已绑定"

#: floating/settings.py:19 floating/settings.py:31
msgid "Floating Binding"
msgstr "绑定中"

#: floating/settings.py:20 floating/settings.py:32
msgid "Floating Released"
msgstr "已释放"

#: floating/settings.py:21 floating/settings.py:33
msgid "Floating Releasing"
msgstr "释放中"

#: floating/settings.py:44
msgid "allocate"
msgstr "申请"

#: floating/settings.py:45
msgid "release"
msgstr "释放"

#: floating/settings.py:46
msgid "associate"
msgstr "绑定云主机"

#: floating/settings.py:47
msgid "disassociate"
msgstr "解绑云主机"

#: forum/models.py:7
msgid "title"
msgstr "标题"

#: forum/models.py:8 forum/models.py:26
msgid "content"
msgstr "内容"

#: forum/models.py:13
msgid "Close Status"
msgstr "关闭"

#: forum/views.py:29 forum/views.py:30
msgid "Creating Forum error"
msgstr "创建工单失败"

#: forum/views.py:46
msgid "Creating Forum reply success"
msgstr "回复成功"

#: forum/views.py:50
msgid "Creating Forum reply error"
msgstr "回复失败"

#: forum/views.py:64
msgid "Close forum success"
msgstr "关闭工单成功"

#: forum/views.py:66
msgid "No selected forum"
msgstr "选择工单"

#: forum/views.py:69
msgid "Close Forum error"
msgstr "关闭工单失败"

#: idc/models.py:15
msgid "openstack host"
msgstr ""

#: idc/models.py:15
msgid "IP of Compute Center"
msgstr ""

#: idc/models.py:16
msgid "default project"
msgstr "默认项目"

#: idc/models.py:17
msgid "Project Name of Data Center,recommended: admin"
msgstr ""

#: idc/models.py:18
msgid "default project user"
msgstr "默认用户"

#: idc/models.py:19
msgid "User who can visit the project"
msgstr ""

#: idc/models.py:20
msgid "default user password"
msgstr "默认密码"

#: idc/models.py:21
msgid "usually http://host:5000/v2.0"
msgstr ""

#: idc/models.py:22
msgid "External Network Name"
msgstr "外部网络名称"

#: idc/models.py:36 idc/models.py:37
msgid "DataCenter"
msgstr "数据中心"

#: idc/models.py:49
msgid "Tenant"
msgstr "项目"

#: idc/models.py:50
msgid "Tenant UUID"
msgstr ""

#: idc/models.py:51
msgid "User"
msgstr ""

#: idc/models.py:52
msgid "Password"
msgstr ""

#: idc/models.py:59 idc/models.py:60
msgid "UserDataCenter"
msgstr "用户数据中心"

#: idc/views.py:68
msgid "Data Center is created successfully!"
msgstr "数据中心创建成功！"

#: idc/views.py:72
msgid "Data Center data is not valid!"
msgstr "提交的数据有错误！"

#: idc/views.py:77 idc/views.py:104
msgid "Failed to create data center for unknown reason."
msgstr "未知错误导致数据中心创建失败．"

#: idc/views.py:89
msgid "This host has been used by other data center."
msgstr "该节点ＩＰ已被其他数据中心使用."

#: idc/views.py:100
msgid "Data Center is updated successfully!"
msgstr "数据中心更新成功！"

#: idc/views.py:115
msgid "Data centers have been deleted!"
msgstr "数据中心删除成功！"

#: idc/views.py:119
msgid "Failed to delete data centers for unknown reason."
msgstr "数据中心删除失败"

#: image/models.py:6
msgid "Windows"
msgstr "Windows"

#: image/models.py:7
msgid "Linux"
msgstr "Linux"

#: image/models.py:17
msgid "Image Name"
msgstr "名称"

#: image/models.py:18
msgid "UUID"
msgstr "UUID"

#: image/models.py:18
msgid "Openstack Image UUID"
msgstr "Openstack 镜像 UUID"

#: image/models.py:19
msgid "Login Name"
msgstr "登录用户名"

#: image/models.py:20
msgid "Image OS Type"
msgstr "镜像类型"

#: image/models.py:45 image/models.py:46
msgid "Image"
msgstr "镜像 "

#: image/views.py:45
msgid "Image is created successfully!"
msgstr "镜像创建成功"

#: image/views.py:48 image/views.py:69
msgid "Image data is not valid!"
msgstr "镜像数据有错误！"

#: image/views.py:53
msgid "Failed to create image for unknown reason."
msgstr "镜像创建失败"

#: image/views.py:66
msgid "Image is updated successfully!"
msgstr "镜像更新成功"

#: image/views.py:74
msgid "Failed to update image for unknown reason."
msgstr "镜像更新失败"

#: image/views.py:84
msgid "Images have been deleted!"
msgstr "镜像已删除"

#: instance/models.py:15
msgid "Linux Image"
msgstr "Linux系统"

#: instance/models.py:17
msgid "Windows Image"
msgstr "Windows系统"

#: instance/models.py:23
msgid "Instance Name"
msgstr ""

#: instance/models.py:27
msgid "Terminate Date"
msgstr ""

#: instance/models.py:28 instance/models.py:79
msgid "Cpu Cores"
msgstr ""

#: instance/models.py:29
msgid "Memory"
msgstr ""

#: instance/models.py:29
msgid "System Disk"
msgstr ""

#: instance/models.py:31
msgid "OS FlavorID"
msgstr ""

#: instance/models.py:42
msgid "Private IP"
msgstr ""

#: instance/models.py:82
msgid "Memory MB"
msgstr ""

#: instance/models.py:83
msgid "Price"
msgstr ""

#: instance/models.py:89 instance/models.py:90
msgid "Flavor"
msgstr "规格"

#: instance/settings.py:24 instance/settings.py:47
msgid "Instance Waiting"
msgstr "等待调度"

#: instance/settings.py:25 instance/settings.py:48
msgid "Instance Running"
msgstr "运行中"

#: instance/settings.py:26 instance/settings.py:49
msgid "Instance Booting"
msgstr "启动中"

#: instance/settings.py:27 instance/settings.py:50
msgid "Instance Rebooting"
msgstr "重启中"

#: instance/settings.py:28 instance/settings.py:51
msgid "Instance Paused"
msgstr "暂停中"

#: instance/settings.py:29 instance/settings.py:52 instance/settings.py:80
#: instance/settings.py:103
msgid "Instance Power Off"
msgstr "已关机"

#: instance/settings.py:30 instance/settings.py:53
msgid "Instance Shutting Down"
msgstr "正在关机"

#: instance/settings.py:31 instance/settings.py:54
msgid "Instance Deploying"
msgstr "部署中"

#: instance/settings.py:32 instance/settings.py:55
msgid "Instance Deploy Failed"
msgstr "部署失败"

#: instance/settings.py:33 instance/settings.py:56
msgid "Instance Locked"
msgstr "已锁定"

#: instance/settings.py:34 instance/settings.py:57
msgid "Instance Delete"
msgstr "已删除"

#: instance/settings.py:35 instance/settings.py:58
msgid "Instance Error"
msgstr "错误"

#: instance/settings.py:36 instance/settings.py:59
msgid "Instance Backuping"
msgstr "备份中"

#: instance/settings.py:37 instance/settings.py:60
msgid "Instance Restoring"
msgstr "恢复中"

#: instance/settings.py:38 instance/settings.py:61
msgid "Instance Resizing"
msgstr "调整规格"

#: instance/settings.py:39 instance/settings.py:62
msgid "Instance Expired"
msgstr "已过期"

#: instance/settings.py:40 instance/settings.py:63
msgid "Instance Deleting"
msgstr "删除中"

#: instance/settings.py:78
msgid "Instance Active"
msgstr "开机"

#: instance/settings.py:79
msgid "Instance Build"
msgstr "构建"

#: instance/settings.py:81
msgid "Instance Pause"
msgstr "暂停"

#: instance/settings.py:82
msgid "Instance Lock"
msgstr "锁定"

#: instance/settings.py:83
msgid "Instance Resize"
msgstr "调整规则"

#: instance/settings.py:84 instance/settings.py:101
msgid "Instance Reboot"
msgstr "重启"

#: instance/settings.py:85
msgid "Instance Restore"
msgstr "恢复"

#: instance/settings.py:102
msgid "Instance Power On"
msgstr "开机"

#: instance/settings.py:104
msgid "Instance VNC"
msgstr "VNC远程桌面"

#: instance/settings.py:105
msgid "Instance Bind Floating"
msgstr "绑定公网IP"

#: instance/settings.py:106
msgid "Instance Unbind Floating"
msgstr "解绑公网IP"

#: instance/settings.py:107
msgid "Instance Change Firewall"
msgstr "修改防火墙"

#: instance/settings.py:108
msgid "Instance Attach Volume"
msgstr "挂载磁盘"

#: instance/settings.py:109
msgid "Instance Detach Volume"
msgstr "卸载磁盘"

#: instance/settings.py:110
msgid "Instance Terminate"
msgstr "删除"

#: instance/settings.py:111
msgid "Instance Launch"
msgstr "创建云主机"

#: instance/settings.py:157
msgid "Second"
msgstr "秒"

#: instance/settings.py:158
msgid "Minute"
msgstr "分"

#: instance/settings.py:159
msgid "Hourly"
msgstr "时"

#: instance/settings.py:160
msgid "Daily"
msgstr "天"

#: instance/settings.py:161
msgid "Weekly"
msgstr "周"

#: instance/settings.py:162
msgid "Monthly"
msgstr "月"

#: instance/settings.py:163
msgid "Yearly"
msgstr "年"

#: instance/views.py:60
msgid "Flavor is created successfully!"
msgstr "规格创建成功！"

#: instance/views.py:63 instance/views.py:82
msgid "Flavor data is not valid!"
msgstr "规格数据有错误！"

#: instance/views.py:68
msgid "Failed to create flavor for unknown reason."
msgstr "未知错误导致规格创建失败．"

#: instance/views.py:79
msgid "Flavor is updated successfully!"
msgstr "规格更新成功"

#: instance/views.py:87
msgid "Failed to update flavor for unknown reason."
msgstr "规格更新失败"

#: instance/views.py:94
msgid "Flavors have been deleted!"
msgstr "规格已删除"

#: lbaas/models.py:11
msgid "Pool UUID"
msgstr "资源池ID"

#: lbaas/models.py:12 lbaas/models.py:60
msgid "Pool name"
msgstr "名称"

#: lbaas/models.py:13 lbaas/models.py:61
msgid "Description"
msgstr "描述"

#: lbaas/models.py:15
msgid "protocol"
msgstr "协议"

#: lbaas/models.py:16
msgid "Lb method"
msgstr "负载策略"

#: lbaas/models.py:17
msgid "Provider"
msgstr ""

#: lbaas/models.py:18 lbaas/models.py:42 lbaas/models.py:72
#: lbaas/models.py:101
msgid "Admin state up"
msgstr ""

#: lbaas/models.py:29 lbaas/models.py:30
msgid "BalancerPool"
msgstr "负载均衡器"

#: lbaas/models.py:38
msgid "Member_UUID"
msgstr ""

#: lbaas/models.py:39 lbaas/models.py:64 lbaas/models.py:65
msgid "Address"
msgstr "地址"

#: lbaas/models.py:40 lbaas/models.py:67
msgid "Protocol port"
msgstr "端口"

#: lbaas/models.py:41
msgid "Weight"
msgstr "权重"

#: lbaas/models.py:52 lbaas/models.py:53
msgid "BalancerMember"
msgstr "负载均衡器成员"

#: lbaas/models.py:58
msgid "VIP_uuid"
msgstr "虚IP ID"

#: lbaas/models.py:69
msgid "Port id"
msgstr "端口"

#: lbaas/models.py:70
msgid "Session persistence"
msgstr "会话持久化"

#: lbaas/models.py:71
msgid "Connection limit"
msgstr "链接限制"

#: lbaas/models.py:82 lbaas/models.py:83
msgid "BalancerVIP"
msgstr ""

#: lbaas/models.py:92
msgid "Monitor uuid"
msgstr "监听器"

#: lbaas/models.py:93
msgid "Monitor name"
msgstr "名称"

#: lbaas/models.py:94
msgid "Type"
msgstr "类型"

#: lbaas/models.py:95
msgid "Delay"
msgstr "延迟"

#: lbaas/models.py:96
msgid "Timeout"
msgstr "超时"

#: lbaas/models.py:97
msgid "Max retries"
msgstr "最大重试次数"

#: lbaas/models.py:98
msgid "Http method"
msgstr "HTTP Method"

#: lbaas/models.py:99
msgid "Url path"
msgstr ""

#: lbaas/models.py:100
msgid "Expected codes"
msgstr "期望状态吗"

#: lbaas/models.py:118 lbaas/models.py:119
msgid "BalancerMonitor"
msgstr "负载均衡监听器"

#: lbaas/models.py:129 lbaas/models.py:130
msgid "BalancerPoolMonitor"
msgstr "负载均衡器不存在"

#: lbaas/settings.py:15 network/settings.py:19 network/settings.py:32
msgid "CREATING"
msgstr "创建中"

#: lbaas/settings.py:16 lbaas/settings.py:26 network/settings.py:16
#: network/settings.py:29
msgid "ACTIVE"
msgstr "已启动"

#: lbaas/settings.py:17 lbaas/settings.py:27
msgid "CREATED"
msgstr "创建成功"

#: lbaas/settings.py:18 lbaas/settings.py:28 network/settings.py:17
#: network/settings.py:30
msgid "DOWN"
msgstr "已关闭"

#: lbaas/settings.py:19 lbaas/settings.py:29 network/settings.py:18
#: network/settings.py:31
msgid "ERROR"
msgstr "错误"

#: lbaas/settings.py:20 lbaas/settings.py:30
msgid "INACTIVE"
msgstr "未激活"

#: lbaas/settings.py:21 lbaas/settings.py:31 network/settings.py:21
#: network/settings.py:34
msgid "UPDATING"
msgstr "更新中"

#: lbaas/settings.py:22 lbaas/settings.py:32 network/settings.py:20
#: network/settings.py:33
msgid "DELETING"
msgstr "删除中"

#: lbaas/settings.py:25
msgid "Creating"
msgstr "创建中"

#: lbaas/views.py:41
msgid "Balancer no exists"
msgstr "负载均衡器不存在"

#: lbaas/views.py:58
msgid "Create balancer pool success"
msgstr "创建负载均衡器成功"

#: lbaas/views.py:73
msgid "Update balancer pool success"
msgstr "更新负载均衡器成功"

#: lbaas/views.py:76
msgid "Selected balancer no exist"
msgstr "负载均衡器不存在"

#: lbaas/views.py:78
msgid "Balancer data valid no pass"
msgstr "负载均衡器数据有误"

#: lbaas/views.py:90 lbaas/views.py:95
msgid "Delete balancer pool"
msgstr "删除负载均衡器"

#: lbaas/views.py:97
msgid "Balancer pool no exist"
msgstr "负载均衡器不存在"

#: lbaas/views.py:108
msgid "No select balancer pool"
msgstr "请选择负载均衡器"

#: lbaas/views.py:119
msgid "Create balancer vip success"
msgstr "创建负载均衡器成功"

#: lbaas/views.py:132
msgid "Vip update success"
msgstr "虚IP更新成功"

#: lbaas/views.py:134
msgid "Vip update fail"
msgstr "虚IP更新失败"

#: lbaas/views.py:137
msgid "Vip no exist"
msgstr "虚IP不存在"

#: lbaas/views.py:139
msgid "Vip data valid no pass"
msgstr "虚IP数据有误"

#: lbaas/views.py:155
msgid "Delete balancer vip"
msgstr "删除虚IP"

#: lbaas/views.py:157
msgid "Balancer pool or vip no exits"
msgstr "负载均衡器或者VIP不存在"

#: lbaas/views.py:166
msgid "No select float ip or vip"
msgstr "选择公网IP或者VIP"

#: lbaas/views.py:177
msgid "Vip association floating success"
msgstr "虚IP关联公网IP成功"

#: lbaas/views.py:179
msgid "Vip association floating fail"
msgstr "虚IP关联公网IP失败"

#: lbaas/views.py:186
msgid "Vip disassociation floating success"
msgstr "虚IP解除关联公网IP成功"

#: lbaas/views.py:188
msgid "Vip disassociation floating fail"
msgstr "虚IP解除关联公网IP失败"

#: lbaas/views.py:191
msgid "Vip or Floating no exist"
msgstr "虚IP或者公网IP不存在"

#: lbaas/views.py:200
msgid "No select balancer or monitor"
msgstr "选择负载均衡器和监听器"

#: lbaas/views.py:211
msgid "Balancer monitor association success"
msgstr "监听器管理成功"

#: lbaas/views.py:213
msgid "Balancer monitor association fail"
msgstr "监听器关联失败"

#: lbaas/views.py:222
msgid "Balancer monitor cancel association success"
msgstr "监听器解除成功"

#: lbaas/views.py:224
msgid "Balancer monitor cancel association fail"
msgstr "监听器解除失败"

#: lbaas/views.py:227
msgid "balancer or monitor no exist"
msgstr "负载均衡器或者监听器不存在"

#: lbaas/views.py:245
msgid "Selected balancer no exists"
msgstr "请选择负载均衡器"

#: lbaas/views.py:261
msgid "Add member success"
msgstr "添加成员成功"

#: lbaas/views.py:263
msgid "No select member"
msgstr "请选择成员"

#: lbaas/views.py:267
msgid "Member no exists"
msgstr "成员不存在"

#: lbaas/views.py:273
msgid "Member update success"
msgstr "成员信息更新成功"

#: lbaas/views.py:275
msgid "Update member fail"
msgstr "成员信息更新失败"

#: lbaas/views.py:277
msgid "Member data valid no pass"
msgstr "成员信息数据有误"

#: lbaas/views.py:293
msgid "Delete balancer member success"
msgstr "删除成员成功"

#: lbaas/views.py:295
msgid "Balancer member no exist"
msgstr "成员不存在"

#: lbaas/views.py:321
msgid "Create balancer monitor success"
msgstr "创建监听器成功"

#: lbaas/views.py:324
msgid "Create balancer monitor Fail"
msgstr "创建监听器失败"

#: lbaas/views.py:328
msgid "Monitor no exists"
msgstr "监听器不存在"

#: lbaas/views.py:336
msgid "Update balancer monitor success"
msgstr "更新监听器成功"

#: lbaas/views.py:338
msgid "Update balancer monitor Fail"
msgstr "更新监听器失败"

#: lbaas/views.py:340
msgid "Monitor data valid no pass"
msgstr "监听器数据有误"

#: lbaas/views.py:349
msgid "Monitor is used,can't deleted"
msgstr "使用中的监听器不允许删除"

#: lbaas/views.py:355
msgid "Delete balancer monitor success"
msgstr "删除监听器成功"

#: lbaas/views.py:357
msgid "Balancer monitor no exist"
msgstr "监听器不存在"

#: network/models.py:9 network/models.py:46
msgid "Network Name"
msgstr "名称"

#: network/models.py:10
msgid "OS Network UUID"
msgstr ""

#: network/models.py:48
msgid "OS Subnet UUID"
msgstr ""

#: network/models.py:50
msgid "IPv4 CIDR"
msgstr ""

#: network/models.py:51
msgid "IP Version"
msgstr "IP 版本"

#: network/models.py:67
msgid "Router Name"
msgstr "名称"

#: network/models.py:68 network/models.py:69
msgid "OS Router UUID"
msgstr "Router UUID"

#: network/models.py:70
msgid "Whether to open the gateway"
msgstr ""

#: network/models.py:86
msgid "Port"
msgstr "端口"

#: network/models.py:97 network/models.py:98
msgid "RouterInterface"
msgstr "路由器端口"

#: network/settings.py:15 network/settings.py:28
msgid "BUILD"
msgstr "构建中"

#: network/views.py:56
msgid "Create network success"
msgstr "创建私有网络成功"

#: network/views.py:61
msgid "Create network error"
msgstr "创建私有网络失败"

#: network/views.py:67
msgid "The selected network not exist"
msgstr "私有网络不存在"

#: network/views.py:72
msgid "Update network success"
msgstr "更新私有网络成功"

#: network/views.py:82
msgid "Default network can not be deleted"
msgstr "默认的私有网络不允许删除"

#: network/views.py:85
msgid "Was unable to delete the use of the network"
msgstr "使用中的私有网络不允许删除"

#: network/views.py:89 network/views.py:96
msgid "Network deleted success"
msgstr "私有网络删除成功"

#: network/views.py:85
msgid "The selected network does not exist"
msgstr "私有网络不存在"

#: network/views.py:106
msgid "Default network not allow operation"
msgstr "默认私有网络不支持此操作"

#: network/views.py:125 network/views.py:131
#, fuzzy
#| msgid "Unknown volume action"
msgid "Unkown operation"
msgstr "未知的磁盘操作"

#: network/views.py:141
msgid "Unkown router"
msgstr ""

#: network/views.py:42
#, fuzzy
#| msgid "Network deleted success"
msgid "Network address exists"
msgstr "私有网络删除成功"

#: network/views.py:101
#, fuzzy
#| msgid "Network deleted success"
msgid "Network deleted error"
msgstr "私有网络删除成功"

#: network/views.py:125 network/views.py:131
#, fuzzy
#| msgid "Unknown volume action"
msgid "Unkown operation"
msgstr "未知的磁盘操作"

#: network/views.py:141
msgid "Unkown router"
msgstr ""

#: network/views.py:144
#: network/views.py:111
msgid "network exist address , not operation"
msgstr "子网地址已经存在"

#: network/views.py:160
msgid "Link router success"
msgstr "加入路由器成功"

#: network/views.py:166
#, fuzzy
#| msgid "Link router success"
msgid "Link router error"
msgstr "加入路由器成功"

#: network/views.py:141
msgid "Network not add subnet ,not operation "
msgstr "请将私有网络先加入路由器"

#: network/views.py:159 network/views.py:163
msgid "Network operation error"
msgstr "操作异常"

#: network/views.py:160
msgid "Leave router success "
msgstr "离开路由器成功"

#: network/views.py:220
msgid "Create router success"
msgstr "创建路由器成功"

#: network/views.py:222
msgid "Valid Router fail"
msgstr "数据提交异常"

#: network/views.py:225
msgid "Create router fail"
msgstr "创建路由器失败"

#: network/views.py:229
msgid "The selected router not exist"
msgstr "路由器不存在"

#: network/views.py:243
msgid "Update router success"
msgstr "路由器更新成功"

#: network/views.py:253
msgid "Default Router can not be deleted"
msgstr "默认路由器不允许删除"

#: network/views.py:258
msgid "Router used can not deleted"
msgstr "使用中的路由器不允许删除"

#: network/views.py:263
msgid "Router deleted success"
msgstr "路由器删除成功"

#: network/views.py:265
msgid "The selected router does not exist"
msgstr "路由器不存在"

#: overview/views.py:43
msgid "The host IP is used by other Data Center, Please check your host IP."
msgstr "该节点ＩＰ已被其他数据中心使用，请检查您的节点IP."

#: overview/views.py:48
msgid "Unknown Error happened when creating data center!"
msgstr "未知错误导致数据中心创建失败！"

#: overview/views.py:55
msgid "Data center is initialized successfully!"
msgstr "数据中心初始化成功"

#: overview/views.py:75 overview/views.py:101
msgid "Unknown Error happened when creating flavors!"
msgstr "创建规格失败"

#: overview/views.py:78 overview/views.py:104
msgid "Flavors are initialized successfully!"
msgstr "规格更新成功"

#: volume/models.py:10
msgid "Volume name"
msgstr "名称"

#: volume/models.py:11
msgid "OS Volume UUID"
msgstr "OS UUID"

#: volume/models.py:13
msgid "Volume size"
msgstr "磁盘容量"

#: volume/models.py:14
msgid "Volume Type"
msgstr "磁盘类型"

#: volume/settings.py:21 volume/settings.py:40
msgid "Volume Creating"
msgstr "创建中"

#: volume/settings.py:22 volume/settings.py:41
msgid "Volume Attaching"
msgstr "挂载中"

#: volume/settings.py:23 volume/settings.py:42
msgid "Volume Available"
msgstr "可用"

#: volume/settings.py:24 volume/settings.py:43
msgid "Volume Backing_up"
msgstr ""

#: volume/settings.py:25 volume/settings.py:44
msgid "Volume Deleting"
msgstr "删除中"

#: volume/settings.py:26 volume/settings.py:45
msgid "Volume Downloading"
msgstr "下载中"

#: volume/settings.py:27 volume/settings.py:46
msgid "Volume Error"
msgstr "错误"

#: volume/settings.py:28 volume/settings.py:47
msgid "Volume Error_deleting"
msgstr "删除失败"

#: volume/settings.py:29 volume/settings.py:48
msgid "Volume Error_restoring"
msgstr ""

#: volume/settings.py:30 volume/settings.py:49
msgid "Volume In Use"
msgstr "使用中"

#: volume/settings.py:31 volume/settings.py:50
msgid "Volume Restoring_backup"
msgstr ""

#: volume/settings.py:32 volume/settings.py:51
msgid "Volume Unrecognized"
msgstr "未知"

#: volume/settings.py:33 volume/settings.py:52
msgid "Volume uploading"
msgstr "更新中"

#: volume/settings.py:57
msgid "Capacity"
msgstr "容量型"

#: volume/settings.py:58
msgid "Performance"
msgstr "性能型"

#: volume/views.py:56
msgid "Creating Volume"
msgstr "创建中"

#: volume/views.py:61 volume/views.py:66
#: volume/views.py:58 volume/views.py:65
msgid "Volume create error"
msgstr "创建磁盘成功"

#: volume/views.py:78
msgid "Volume update success"
msgstr "更新磁盘成功"

#: volume/views.py:80
msgid "No select Volume "
msgstr "请选择磁盘"

#: volume/views.py:83
msgid "Volume update error"
msgstr "更新磁盘失败"

#: volume/views.py:95
msgid "Unknown volume action"
msgstr "未知的磁盘操作"

#: volume/views.py:98
msgid "Volume action error"
msgstr "操作失败"

#: volume/views.py:114
msgid "Attaching volume"
msgstr "挂载中"

#: volume/views.py:119
#, fuzzy
#| msgid "Attaching volume"
msgid "Attach volume error"
msgstr "挂载中"

#: volume/views.py:127
msgid "Detaching volume"
msgstr "卸载中"

#: volume/views.py:132
#, fuzzy
#| msgid "Deleting Volume error"
msgid "Detach volume error"
msgstr "删除磁盘失败"

#: volume/views.py:137
#: volume/views.py:124
#, python-format
msgid "Volume attached to instance,instance:%(instance)s"
msgstr "磁盘已挂载到云主机:%(instance)s"

#: volume/views.py:145 volume/views.py:154
msgid "Deleting Volume"
msgstr "删除中"

#: volume/views.py:150
#: volume/views.py:136
msgid "Deleting Volume error"
msgstr "删除磁盘失败"

#~ msgid "Default network not allow operation"
#~ msgstr "默认私有网络不支持此操作"

#~ msgid "Network not add subnet ,not operation "
#~ msgstr "请将私有网络先加入路由器"

#~ msgid "full"
#~ msgstr "全量备份"

#~ msgid "increment"
#~ msgstr "增量备份"

#~ msgid "Wating Backup"
#~ msgstr "等待备份"
